alg/cuCompactor.cuh

```diff
@@ -0,0 +1,165 @@
+/*
+ * cuCompactor.h
+ *
+ *  Created on: 21/mag/2015
+ *      Author: knotman
```

Comment:
do we have permission to use this file here?  I don't see license info with this, and we need to make sure that anything we include is open source and has a license compatible with TECA's license

Reply:
Here is the license info: [https://github.com/knotman90/cuStreamComp/blob/master/LICENSE](url)
Is it compatible with TECA's license?

Reply:
I just removed this file and commented everything related to its functions. Then we can uncomment it in the future and use it.